Incorrect type definition in `astro/client.d.ts` causes TypeScript validation error

### Astro Info

```block
Astro                    v5.1.9
Node                     v22.13.0
System                   macOS (arm64)
Package Manager          npm
Output                   static
Adapter                  none
Integrations             @astrojs/tailwind
```

### If this issue only occurs in one browser, which browser is a problem?

_No response_

### Describe the Bug

Hello! 👋

It seems that the definition of the `TransitionSwapFunctionModule` type inside the [astro/client.d.ts](https://github.com/withastro/astro/blob/821d642753812a83b73300c04ccd2883d481b08b/packages/astro/client.d.ts#L172) file is incorrect. There seems to be an extra comma that should not be in the import statement.

Therefore, when the `tsc --noEmit` command is launched, the validation fails due to the following:

```
example on main *2 !2 ❯ npx tsc --noEmit                                                                                                             arm64 ✘ 2
node_modules/astro/client.d.ts:173:2 - error TS1005: '{' expected.

173  );
     ~

  node_modules/astro/client.d.ts:173:2
    173  );
         ~
    The parser expected to find a '}' to match the '{' token here.


Found 1 error in node_modules/astro/client.d.ts:173
```


### What's the expected result?

The expected result is that the syntax is correct and the `tsc --noEmit` command ends with a successful validation.

Thanks in advance! 🙏

Regards!

### Link to Minimal Reproducible Example

https://github.com/withastro/astro/blob/821d642753812a83b73300c04ccd2883d481b08b/packages/astro/client.d.ts#L172

### Participation

- [ ] I am willing to submit a pull request for this issue.

--- a/.changeset/clean-suns-shout.md
+++ b/.changeset/clean-suns-shout.md
@@ -1,0 +1,5 @@
+---
+'astro': patch
+---
+
+Fixes broken type declaration

--- a/packages/astro/client.d.ts
+++ b/packages/astro/client.d.ts
@@ -168,9 +168,8 @@ declare module 'astro:transitions/client' {
 	export type TransitionBeforeSwapEvent = import('./dist/virtual-modules/transitions-events.js').TransitionBeforeSwapEvent;
 	export const isTransitionBeforePreparationEvent: EventModule['isTransitionBeforePreparationEvent'];
 	export const isTransitionBeforeSwapEvent: EventModule['isTransitionBeforeSwapEvent'];
-	type TransitionSwapFunctionModule = typeof import(
-		'./dist/virtual-modules/transitions-swap-functions.js',
-	);
+	// biome-ignore format: bug
+	type TransitionSwapFunctionModule = typeof import('./dist/virtual-modules/transitions-swap-functions.js');
 	export const swapFunctions: TransitionSwapFunctionModule['swapFunctions'];
 }
 

